packaging/rpm/cvmfs-universal.spec

```diff
@@ -572,18 +595,22 @@ fi
 %{_libdir}/libcvmfs_server.so.%{version}
 %{_libdir}/libcvmfs_server_debug.so
 %{_libdir}/libcvmfs_server_debug.so.%{version}
-%{_sysconfdir}/cvmfs/cvmfs_server_hooks.sh.demo
+%config %{_sysconfdir}/cvmfs/cvmfs_server_hooks.sh.demo
 %dir %{_sysconfdir}/cvmfs/repositories.d
+%dir /var/www/
+/var/www/wsgi-scripts
 /var/www/wsgi-scripts/cvmfs-server/cvmfs-api.wsgi
 /usr/share/cvmfs-server/
 /var/lib/cvmfs-server/
+%dir /var/spool/cvmfs
 /var/spool/cvmfs/README
 %doc COPYING AUTHORS README.md ChangeLog
 
 %files shrinkwrap
 %defattr(-,root,root)
 %{_bindir}/cvmfs_shrinkwrap
-/usr/libexec/cvmfs/shrinkwrap/spec_builder.py*
+%dir /usr/libexec/cvmfs/shrinkwrap
+/usr/libexec/cvmfs/shrinkwrap/spec_builder.py
```

Comment:
I guess we should have written down _why_ we want the bytecode packaged back at the time.  There is an [extra commit](https://github.com/cvmfs/cvmfs/pull/2231/commits/6135728d9d152d91fcf4d65248b880117fb06ce7) for this, so I guess there was a reason. Is it important to leave the byte code out? If it is, we can just see what happens and if we hit the original issue again.

Reply:
I think I just removed the * because I didn't know what it was doing there ... it seems strange to ship pyc files right? python will create them the first time it gets executed. I would really worry if that doesn't work. Lets try.